packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -52,6 +52,16 @@ export enum FilterActions {
     REMOVE = 'remove',
 }
 
+// TODO: add a 'disabled' field to tileConfig.
+// For now we use this empty tile config to indicate that
+// a chart should not get the default filters applied to it.
+// A clearer solution would be to add a 'disabled' field we could
+// toggle to mark the filter as disabled for that tile.
+const DISABLE_FILTER_TILE_CONFIG = {
+    fieldId: '',
+    tableName: '',
+};
```

Comment:
is it going to be a lot of work if we add disabled field to the tileConfig now? im worried that if we change later we should still have a legacy code to support backward compatibility of the config

Reply:
That's a good question. I guess then I wonder: will it be more work later than now? What if we want to add it now? We'd have to add a migration? How hard is that to do?

Reply:
since it's saved as a json I don't think we'll ever write a migration for that. what we can do here is to introduce a type change. in this case is
```tsx
type DashboardFieldTarget = {
    fieldId: string;
    tableName: string;
};
```

we can change this to

```tsx
type DashboardFieldTarget = {
  fieldId: string;
  tableName: string;
} | {
  disabled: true;
}

I think this should work and is a fairly straightforward change